src/aind_metadata_mapper/smartspim/models.py

```diff
@@ -22,5 +34,10 @@ class JobSettings(BaseJobSettings):
     asi_filename: str = "derivatives/ASI_logging.txt"
     mdata_filename_json: str = "derivatives/metadata.json"
 
-    # Metadata provided by microscope operators
-    processing_manifest_path: str = "derivatives/processing_manifest.json"
+    # Fetch info provided by microscope operators in SLIMS
+    processing_manifest_path: Optional[str] = Field(
+        default="derivatives/processing_manifest.json",
+        description=("Deprecated, use metadata_service_path instead. "),
+    )
+    metadata_service_domain: str
```

Comment:
separated `metadata_service_domain` and `metadata_service_path` to match pattern of GatherMetadata JobSettings. However, that might be unnecessary so let me know and I can just make it one with no default, or default to "http://aind-metadata-service/slims/smartspim_imaging